src/workstation-gui/app/models/cookbook_upload.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class CookbookUpload < ApplicationRecord
```

Comment:
curious to understand, why not `class Cookbook < ApplicationRecord`, like it's been done in the PolicyFile model `class Policy < ApplicationRecord`?

Reply:
Changed it to `Cookbook`. This makes more sense as we add new functionality. 
Earlier it was named `CookbookUpload` because it contains only the method which is responsible for uploading cookbook but when we add more method than `CookbookUpload`  doesn't make much sense.